packages/components/autocomplete/src/Autocomplete.tsx

```diff
@@ -0,0 +1,246 @@
+import React, { useState } from 'react';
+import { cx } from 'emotion';
+import { useCombobox } from 'downshift';
+
+import { CommonProps, mergeRefs } from '@contentful/f36-core';
+import {
+  TextInput,
+  TextInputProps,
+  useFormControl,
+} from '@contentful/f36-forms';
+import { IconButton } from '@contentful/f36-button';
+import { CloseIcon, ChevronDownIcon } from '@contentful/f36-icons';
+import { Popover } from '@contentful/f36-popover';
+
+import { getAutocompleteStyles } from './Autocomplete.styles';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export interface AutocompleteProps<ItemType = any>
+  extends CommonProps,
```

Comment:
do we maybe also need props for the size of the dropdown list ?

Reply:
what do you mean by size of the dropdown  list?

Reply:
maxHeight?

Reply:
`maxHeight` and `width`

Reply:
I will add those, but I will give maxHeight a default value because I don't think we should allow huge lists

what I mean is that a list with 20 items and without maxHeight would be 1200px tall 👀